x-pack/test/snapshot_declarations.d.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+declare const expectSnapshot: (
```

Comment:
Please move this to `test/typings` so that it applies to oss as well.

Reply:
done 👍